circle.py

```diff
@@ -2,9 +2,14 @@
 
 
 def area(r):
-    return math.pi * r * r
+    if r >= 0:
+        return math.pi * r * r
+    else:
+        raise ValueError("Size should be > 0.")
```

Comment:
```suggestion
    if r < 0:
        raise ValueError("Size should be > 0.")

    return math.pi * r * r
```

лучше будет как-то так)
!можно не исправлять!